src/main/kotlin/no/nav/familie/tilbake/beregning/TilbakekrevingsberegningService.kt

```diff
@@ -34,6 +36,21 @@ class TilbakekrevingsberegningService(private var kravgrunnlagRepository: Kravgr
                                       private var behandlingRepository: BehandlingRepository,
                                       private var kravgrunnlagsberegningService: KravgrunnlagsberegningService) {
 
+    fun hentBeregningsresultat(behandlingId: UUID): BeregningsresultatDto {
+        val beregningsresultat = beregn(behandlingId)
+        val beregningsresultatsperioder = beregningsresultat.beregningsresultatsperioder.map {
+            BeregningsresultatsperiodeDto(periode = PeriodeDto(it.periode),
+                                          vurdering = it.vurdering,
+                                          feilutbetaltBeløp = it.feilutbetaltBeløp,
+                                          andelAvBeløp = it.andelAvBeløp,
+                                          renteprosent = it.renteprosent,
+                                          tilbakekrevingsbeløp = it.manueltSattTilbakekrevingsbeløp ?: it.tilbakekrevingsbeløp,
```

Comment:
Eg skjønner ikkje heilt denne linja her?
Burde den ikkje alltid hente tilbakekrevingsbeløpet i beregningsresultatet?
Og tilbakekrevingsbeløp = manueltSattTilbakekrevingsbeløp i dei tilfellen kor saksbehandler må ha satt beløpet under saksbehandlingen?
for i dei fleste tilfelle utanom God tro, så blir jo tilbakekrevingsbeløpet rekna ut basert på andelAvBeløp

Reply:
som diskutert endret det til tilbakekrevingsbeløp